### ISSUE TITLE
Abbreviating weekday names further, if space is small

### ISSUE THREAD

**Issue by quite** [alterechtschreibung/MonthCalendarWidget2FOSS#2]
Can the calendar detect what size it is? I suppose so, but I never did any android hacking.

Could it then choose to abbreviate weekday names even more, if needed? On my system, if I have a 2x2 monthcalendar widget, then MON gets wrapped to MO[newline]N, while the other weekdays are displayed fine in one line TUE, WED, etc). And the rest of the calendar looks fine. But, in this case, MO TU WE TH FR SA SU could be chosen and would give a better result.

Ah, I see now that also longer month names, August--February, gets separated from the year, as so: monthname[newline]year, and this causes clipping of the top and the botton of that text. Well, then I suppose months could be abbreviated, as well, when need be. Aug, Sep, Oct, ...

I like the idea of this simple calendar!

**Comment by alterechtschreibung** [comment]
I might work on this, if I find some time. I also have some other projects. If you want you can implement this and I would merge it. What's your phone model?

**Comment by quite** [comment]
Yeah, I would give hacking it myself a go, but I've found that my current day-to-day system is too slow to run a proper Android devel environment. For now, that's a no-go. I use a nxs5.
